rotkehlchen/externalapis/opensea.py

```diff
@@ -240,14 +279,25 @@ def _deserialize_nft(
                     collection = saved_entry
                     if saved_entry.floor_price is not None:
                         floor_price = saved_entry.floor_price
+                        floor_price_asset = saved_entry.floor_price_asset
                 else:  # should not happen. That means collections endpoint doesnt return anything
                     raise DeserializationError(
                         f'Could not find collection {entry["collection"]} in opensea collections '
                         f'endpoint',
                     )
 
-            price_in_eth = max(last_price_in_eth, floor_price)
-            price_in_usd = price_in_eth * eth_usd_price
+            last_price_in_usd = last_price_in_eth * eth_usd_price
+            floor_price_in_usd = floor_price * Inquirer.find_usd_price(asset=floor_price_asset) if floor_price != ZERO else ZERO  # noqa: E501
+
+            if floor_price_in_usd > last_price_in_usd:
```

Comment:
So this logic keeps the highest? It's what we have in the docs right? So we don't need to change anything in the description?

> An estimation of the value of the NFTs you own is counted into your total net worth. The estimation strategy is currently the maximum of either the floor price of the collection or the last sale of the NFT. If a manual price has been given this is always preferred.